app/src/common/components/MovieList/MovieElement/Genres/Genres.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Genres.scss';
+
+export default class Genres extends React.Component {
```

Comment:
Too simple component to be stateful. I would make a functional component from it. Or simply remove it because on the input it has an array.

Reply:
done